lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -119,10 +146,29 @@ public function __construct(Connection $connection, OutputWriter $outputWriter =
             $outputWriter = new OutputWriter();
         }
         $this->outputWriter = $outputWriter;
+        if ($finder === null) {
+            $finder = new RecursiveRegexFinder();
+        }
         $this->migrationFinder = $finder;
     }
 
     /**
+     * @return bool
+     */
+    public function areMigrationsOrganizedByYear()
```

Comment:
Naming seems clear to me here
On Aug 14, 2015 7:28 PM, "mikeSimonson" <notifications@github.com> wrote:

> In lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php
> <https://github.com/doctrine/migrations/pull/333#discussion_r37098611>:
>
> >          $this->migrationFinder = $finder;
> >      }
> >
> >      /**
> > +     * @return bool
> > +     */
> > +    public function areMigrationsOrganizedByYear()
>
> @Ocramius <https://github.com/Ocramius> Do you think the naming is an
> issue here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/migrations/pull/333/files#r37098611>.
>